include/sundials/sundials_context.h

```diff
@@ -36,6 +36,8 @@ SUNDIALS_EXPORT int SUNContext_GetLogger(SUNContext sunctx, SUNLogger* logger);
 SUNDIALS_EXPORT int SUNContext_SetLogger(SUNContext sunctx, SUNLogger logger);
 SUNDIALS_EXPORT int SUNContext_Free(SUNContext* ctx);
 
+static void sun_adiak_helper();
```

Comment:
```suggestion
```

This does not need to. be in the public header file and should go at the top of `sundials_context.c`.